firestore/query.go

```diff
@@ -635,6 +636,13 @@ type filter struct {
 	value     interface{}
 }
 
+// arrayOperators contains the set of FieldFilter operators that require an ArrayValue operand.
```

Comment:
Since this is only used in one place, then move it inside of `toProto()`.

Reply:
Done

Reply:
Switched to direct comparison instead.